ArcGISRuntimeSDKQt_CppSamples/Layers/FeatureCollectionLayerFromPortal/FeatureCollectionLayerFromPortal.cpp

```diff
@@ -0,0 +1,116 @@
+// [WriteFile Name=FeatureCollectionLayerFromPortal, Category=Layers]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "FeatureCollectionLayerFromPortal.h"
+
+#include "FeatureCollection.h"
+#include "FeatureCollectionLayer.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "Portal.h"
+#include "PortalItem.h"
+
+using namespace Esri::ArcGISRuntime;
+
+FeatureCollectionLayerFromPortal::FeatureCollectionLayerFromPortal(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(Basemap::oceans(this), this))
+{
+  const QUrl url("https://www.arcgis.com/");
```

Comment:
this is the default url so you should not need to pass it to the portal. doesn't do any harm though

Reply:
I included this because other SDKs' implementations explicitly declared the url, as well. If there are no strong feelings for/against having it in, I'll keep it just for consistency.